### DISCUSSION TITLE
summarizing existing notes

### DISCUSSION THREAD

**Discussion by sSalvaXx** [nhaouari/obsidian-textgenerator-plugin#60] [Q&A]
Great plugin ! Thank you.
It is amazing for generating new content, however I can't get it right to summarize existing notes. At least, not in a consistent manner.

When I add a last line with the command "summarize this page in 200 words" it provides me with a random new text completely unrelated to the note. 
My understanding is that when the command is provided on the last line, the context should be the whole content but this seems not working as the generated text is unrelated to the content of the note.

However, if I provide the same command then highlight the whole content of the note, **sometimes** I have a relevant summary **but most of the times** I have an error message "check ctrl+shift+i" that does not lead anywhere.

I'd appreciate any suggestion to summarize existing notes.
Many thanks
---SaS

**Comment by sharik14** [comment]
Hello all - I had a similar question.

I've played with placing notes in the textgenerator/prompts folder as hinted at by the settings dialog – but I can't seem to get it to respond or behave as if different prompts have been provided.

Suggestions?

Thanks,
Ben.
